subprojects/core-api/src/main/java/org/gradle/api/artifacts/result/ComponentSelectionReason.java

```diff
@@ -56,6 +53,13 @@
      */
     boolean isExpected();
 
+    /**
+     * Informs whether the component is selected due to being a composite build.
```

Comment:
This comment isn't helpful. It's just the name of the method said differently. When does this occur? What does it mean?